src/components/Avatar/styles.js

```diff
@@ -33,3 +33,30 @@ export const imgWrapper = css`
   justify-content: center;
   align-items: center;
 `
+export const marginBottom = css`
+  margin-bottom: 24px;
+`
+export const avatarMobile = css`
+width: 2rem;
+height: 2rem;
+& @media (min-width: 33.13em) {
+  .c-avatar--mobile {
```

Comment:
This class is not going to be applied anywhere because it is not being used anywhere. Please check this rule again